src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,79 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { Loader } from '../Loader';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const { slug = '' } = useParams();
```

Comment:
You should provide a type for useParams, like `useParams<{ slug: string }>()` to ensure type safety for the slug parameter.